### PR TITLE
add to selfie.md

### PR DESCRIPTION
**Pull request by nstellitano**
To whom it may concern,
   I have attached a change to selfie.md.  I am started this class on its own but have since signed up for the nanodegree program.  Thanks!
Please review this, @casey-collab

Nick

### CODE REVIEW

File: english/selfie/selfie.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
...
\ No newline at end of file
+Oh no! You are made of marshmallow!
+
+This sudden revelation has now complicated the matter.
+
+You hate marshmallows but can you really hate yourself?  
+
+The next logical step then is to: 
+
+[Begin to eat!](eating/inside.md)
```
Comment (casey-collab) [inline english/selfie/selfie.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/selfie/selfie.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
+
+[Build a fire](light-fire/fire.md)
```
Comment (casey-collab) [inline english/selfie/selfie.md]: This link is also invalid.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by nstellitano** [comment]
I have fixed the links.  Thanks.

Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :thumbsup:

**Pull request merged**